drivers/hwinfo/Kconfig

```diff
@@ -150,6 +150,13 @@ config HWINFO_SAM0
 	help
 	  Enable Atmel SAM0 hwinfo driver.
 
+config HWINFO_SMARTBOND
+	bool "Smartbond device ID"
+	default y
+	depends on SOC_FAMILY_SMARTBOND
+	help
+	  Enable Smartbond hwinfo driver.
```

Comment:
Please add a note that only reset cause is supported

Reply:
fixed